sandbox/api/openapi.yaml

```
@@ -1,17 +1,133 @@
+
+    Authentication and authorisation of end users is the responsibility of your application.
+
+    To access this API, use the following security pattern:
+
+    * [Application-restricted RESTful API - signed JWT authentication](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation/application-restricted-restful-apis-signed-jwt-authentication)
+
+    ## Environments and testing
+
+    | Environment | Base URL |
+    |------------ | -------- |
+    | Sandbox     | `https://sandbox.api.service.nhs.uk/nhs-notify-supplier` |
+    | Integration test | `https://int.api.service.nhs.uk/nhs-notify-supplier` |
+    | Production | `https://api.service.nhs.uk/nhs-notify-supplier` |
+
+    ### Sandbox testing
+
+    Our [sandbox environment](https://digital.nhs.uk/developer/guides-and-documentation/testing#sandbox-testing):
+
+    * is for early developer testing
+    * only covers a limited set of scenarios
+    * is stateless, so does not actually persist any updates
+    * is open access, so does not allow you to test authorisation
+
+    For details of sandbox test scenarios, or to try out sandbox using our 'Try this API' feature, see the documentation for each endpoint.
+
+    Alternatively, you can try out the sandbox using our Postman collection
+
+    You can find our postman collection source in our [public repository on github](https://github.com/NHSDigital/nhs-notify-supplier-api/tree/main/postman).
+
+    ### Integration testing
+
+    Our integration test environment:
+
+    * is for formal integration sandbox-testing
+    * is stateful, so persists updates
+    * includes authorisation via [signed JWT authentication](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation/application-restricted-restful-apis-signed-jwt-authentication)
+
+    You need to get your software approved by us before it can go live with this API.
+
+    You will also need to follow our steps to - TBD
+
+    ### Production smoke testing
+
+    Before go-live, you must complete a smoke test in the NHS Notify production environment.
+    The smoke test confirms that your live credentials, connectivity, and print workflow operate correctly end-to-end. It will be carried out in coordination with the NHS Notify Supplier API team.
+    You will retrieve and print one or more live test letters through the production API, send the printed output to the address provided, and submit a Management Information (MI) update for verification.
+    The NHS Notify team will configure your production access, review your results, and confirm that your output meets NHS Notify print specifications.
+
+    ### Onboarding
+
+    You need to get your software approved by us before it can go live with this API.
+    You will also need to be an approved NHS letter supplier under the framework agreement (ADD link) and nominate your technical and operational contacts
```

```suggestion
    You will also need to be an approved NHS letter supplier under the framework agreement (ADD link) and nominate your technical and operational contacts.
```
Also - needs a link?